appyx-navigation/common/src/iosMain/kotlin/com/bumble/appyx/navigation/integration/MainIntegrationPoint.kt

```diff
@@ -0,0 +1,14 @@
+package com.bumble.appyx.navigation.integration
+
+import com.bumble.appyx.navigation.integrationpoint.IntegrationPoint
+
+class MainIntegrationPoint : IntegrationPoint() {
+    override val isChangingConfigurations: Boolean
+        get() = false
+
+    @Suppress("EmptyFunctionBlock")
+    override fun onRootFinished() {}
+
+    @Suppress("EmptyFunctionBlock")
+    override fun handleUpNavigation() {}
```

Comment:
Is there anything we could do here rather than an empty block?

Reply:
Passing `viewController` here and dimissing/popping as required